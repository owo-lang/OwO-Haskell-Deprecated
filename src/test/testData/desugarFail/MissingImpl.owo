module MissingImpl where
 a : Type
