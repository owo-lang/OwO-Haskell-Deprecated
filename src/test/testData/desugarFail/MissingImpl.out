File type: owo
Top module name: .MissingImpl
 postulate a (26 2 2) (27 2 3)
 type
  type universe Type (30 2 6) (34 2 10)
  level: ULevelLit 0
NoImplementationError (Name ((26 2 2 in Just "testData/desugarFail/MissingImpl.owo"), (27 2 3 in Just "testData/desugarFail/MissingImpl.owo")) "a",[],AstTypeLit (Name ((30 2 6 in Just "testData/desugarFail/MissingImpl.owo"), (34 2 10 in Just "testData/desugarFail/MissingImpl.owo")) "Type") (ULevelLit 0))
